clients/cobol-lsp-vscode-extension/src/services/Middleware.ts

```diff
@@ -17,7 +17,7 @@ import {ConfigurationParams, ConfigurationRequest} from "vscode-languageclient";
 import {CopybookDownloadService} from "./copybook/CopybookDownloadService";
 import {CopybookURI} from "./copybook/CopybookURI";
 
-const PARAMS_REGEX = /^([^.]+)\.([^.]+)(\.(quiet|verbose))?\.(.+)\.([^.]+)$/
+const PARAMS_REGEX = /^([^:]+)\:([^:]+)(\:(quiet|verbose))?\:([^:]+)\:([^:]+)\:*([^:]*)$/
```

Comment:
Why do you change the pattern for the params?

Reply:
Because the new parameter is optional and "." can't be used here anymore. There is no way to determine is it a file name with dots or not. When there was a mandatory list of params it was ok, but now it is not possible to stay with "." delimitr.

Reply:
Ok, that makes sense.